pyspark/test/bigdl/test_utils.py

```diff
@@ -260,15 +278,17 @@ def _do_modelTestSingleLayer(self,
                                  input_data,
                                  output_layer,  # a keras layer
                                  functional_api=True,
+                                 random_weights=True,
                                  dump_weights=False,
                                  is_training=False,
-                                 rtol=1e-7,
-                                 atol=1e-7):
+                                 rtol=1e-6,
```

Comment:
why downgrade the precision?

Reply:
Sometimes for some layers (at least 5), after using random weights, we can't get 1e-7.

Reply:
In that case, we can specify the precision for those test separately.

Reply:
OK